docs/library/spi_engine/spi_engine_interconnect.rst

```diff
@@ -54,20 +54,22 @@ Signal and Interface Pins
    * - resetn
      - Synchronous active-low reset.
        Resets the internal state of the module.
-   * - s*_ctrl
-     - :ref:`spi_engine control-interface` slave.
-       | ``s0_ctrl`` connects to the first control interface master.
-       | ``s1_ctrl`` connects to the second control interface master.
+   * - s0_ctrl
+     - | :ref:`spi_engine control-interface` subordinate.
+       | Connects to the first control interface controller.
+   * - s1_ctrl
+     - | :ref:`spi_engine control-interface` subordinate.
+       | Connects to the second control interface controller.
    * - m_ctrl
-     - | :ref:`spi_engine control-interface` master.
-       | Connects to the control interface slave.
+     - | :ref:`spi_engine control-interface` controller.
+       | Connects to the control interface subordinate.
```

Comment:
This doesn't show if the IP-XACT is not compiled (should be shown as a fallback, right?)

Reply:
Opened issue to tack, thanks!
https://github.com/analogdevicesinc/doctools/issues/9